firmware/library/L1_Drivers/onboardled.hpp

```diff
@@ -18,3 +20,63 @@ class OnBoardLedInterface {
            virtual void LedSetAll(uint8_t value)                     = 0;
            virtual uint8_t LedGetValues(void)                        = 0;
 };
+
+class OnBoardLed: public OnBoardLedInterface
+{
+ public:
+  // Initialization involes setting the appropriate pins to be GPIO, and
+  // then outputs. Once they are configured as such, they can be maniputlated
+  // as expected.
```

Comment:
This comment is unnecessary since the code speaks for itself. The user can easily read what the driver is doing from the code below.

Reply:
Got it.